component-library/src/app/components/code-viewer/code-viewer.component.html

```diff
@@ -0,0 +1,27 @@
+<div
+  [id]="config.id"
+  class="code-viewer"
+>
+  <ircc-cl-lib-tabs
+    class="tabs-component"
+    [config]="tabconfig"
+    (valueChange)="getSelectedTab($event)"
+  ></ircc-cl-lib-tabs>
+  <app-accordian-container [open]="openAccordion">
+    <ircc-cl-lib-button
```

Comment:
Ah, I assume this allows you to put a button on one side or the other of the accordion.